src/client/hooks/usePageChange.ts

```diff
@@ -126,7 +135,7 @@ function getCustomTrackingUrl(
       return pathname;
     }
 
-    return CustomTrackingUrls[route](matchResult, trackingConfig);
+    return CustomTrackingUrls[route]!(matchResult, trackingConfig);
```

Comment:
```suggestion
    return CustomTrackingUrls[route]?.(matchResult, trackingConfig) ?? pathname;
```

Reply:
Het kan niet voorkomen dat `pathname` het resultaat is op dit punt. De key `route` is sowieso een bestaande key van `CustomTrackingUrls` als de code hier komt.

Reply:
Dat klopt idd. Maar ik denk dat het beter is de `bang!` te vermijden. Misschien dit?

```
  if (route) {
    const matchResult = matchPath(pathname, route);
    const trackingUrlFn = CustomTrackingUrls[route];

    if (!matchResult || !matchResult.isExact || !trackingUrlFn) {
      return pathname;
    }

    return trackingUrlFn(matchResult, trackingConfig);
  }
```